ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -415,22 +452,16 @@ main()
 	if [ -z "${9}" ]; then
 		MotionRegression="${G_DEFAULT_MOTION_REGRESSION}"
 	else
-		MotionRegression="${9}"
+		MotionRegression=$(interpret_as_bool "${9}")
+	fi
+
+	local DeleteIntermediates
+	if [ -z "${10}" ]; then
+		DeleteIntermediates="${G_DEFAULT_DELETE_INTERMEDIATES}"
```

Comment:
Ditto -- if using the defaults, `DeleteIntermediates` will be `FALSE`, rather than a boolean, which doesn't work with the later code.